strings-csharp/HexToByteArray/HexToByteArray/ConvertHex.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace HexToByteArray
+{
+    public class ConvertHex
+    {
+        public byte[] FromHexString(string hex)
+        {
+            try
+            {
+                return Convert.FromHexString(hex);
+            }
+            catch (Exception)
+            {
+
+                throw;
```

Comment:
If you are just catching an exception so you can rethrow it, you should just remove the try/catch blockk

Reply:
Done. Thanks!